### ISSUE TITLE
[generation] 필수 단어 포함 문장 생성

### ISSUE THREAD

**Issue by gowun** [sktaiflow/message-generator#23]
- 꼭 포함되어야 하는 단어가 문장안에 들어가게 조정하는 방법?
- 꼭 포함되어야 하는 단어가 특정 위치에 들어가게 조정하는 방법?

search 필요... decoder 조정 부분일 수 있음..

**Comment by gowun** [comment]
https://github.com/huggingface/transformers/issues/3910

**Comment by gowun** [comment]
@haven-jeon  @soeque1 

"꼭 포함되어야만 하는 단어를 포함한 문장 생성 방법"에 대해서 이전 미팅 시, 이런 방법이 존재한다고 말씀해 주셨는데... 제가 못 찾아서 그런데 혹시 참고할 만한 코드가 있나요?

지금 드는 생각은 https://github.com/huggingface/transformers/blob/55fe80d0843d2c6c57a09c3ad9e87bac890d1ff4/src/transformers/generation_utils.py#L616 코드에서 beam_search 함수내의 
![image](https://user-images.githubusercontent.com/28796327/109101861-e8bef700-776a-11eb-8353-cfb1ea514dc9.png)
1638의 next_token_logits = outputs.logits[:, -1, :] 이전에 필수포함단어의 토큰 자리에 임의로 1.0을 넣는다던지로 변경할 수 있을 거 같긴 한데요. (토큰열 중 특정 위치에만 넣을지 전체에 다 넣을지는 잘 모르겠습니다 ㅠ) 이런 방법 말고, 간단하게 컨트롤 가능한 방법이 있을까요?
